$OpenBSD: patch-src_network_c,v 1.1 2012/06/14 13:39:44 ajacoutot Exp $

Correct listener logic when no TFTP and no-interface listen address.

--- src/network.c.orig	Thu Jun 14 01:03:06 2012
+++ src/network.c	Thu Jun 14 01:03:29 2012
@@ -546,7 +546,7 @@ void create_bound_listeners(int dienow)
 
   for (if_tmp = daemon->if_addrs; if_tmp; if_tmp = if_tmp->next)
     if (!if_tmp->used && 
-	(new = create_listeners(&if_tmp->addr, 1, dienow)))
+	(new = create_listeners(&if_tmp->addr, daemon->tftp_unlimited, dienow)))
       {
 	new->iface = NULL;
 	new->next = daemon->listeners;
